File: ai-backend/common/enums/llm_model_type_enum.py
"""
LLM 模型类型枚举
"""
from enum import IntEnum


class LlmModelTypeEnum(IntEnum):
    """LLM 模型类型枚举"""
    CHAT = 1  # chat模型
    EMBEDDING = 2  # embedding模型
    RERANK = 3  # rerank模型

File: backend/src/main/java/top/smartduck/ducktodo/common/enums/LlmModelTypeEnum.java
package top.smartduck.ducktodo.common.enums;

import com.baomidou.mybatisplus.annotation.EnumValue;
import com.fasterxml.jackson.annotation.JsonValue;
import lombok.AllArgsConstructor;
import lombok.Getter;

/**
 * LLM 模型类型枚举
 */
@Getter
@AllArgsConstructor
public enum LlmModelTypeEnum {

    CHAT(1, "chat模型"),
    EMBEDDING(2, "embedding模型"),
    RERANK(3, "rerank模型");

    @EnumValue
    @JsonValue
    private final Integer code;
    private final String desc;
}
